test/helpers/create-server.js

```diff
@@ -3,6 +3,8 @@ const bodyParser = require('body-parser');
 const jwt = require('express-jwt');
 const forestExpress = require('../../src');
 
+const JWT_ALGORITHM = process.env.JWT_ALGORITHM || 'HS256';
```

Comment:
[Code duplication] This declaration is made twice in the code.
This never a good sign in my opinion.
Can't we factorise the JWT middleware initialisation somehow?

Reply:
One occurence is in the `src/index.js` file while the other is in `test/helpers/create-server.js`.
I'm not really concerned with this specific code duplication.

I agree that the `init`method complexity in `src/index.js` is to high and that it makes it hard to understand and test but I'm not sure we should refactor it as part of this security issue.

Reply:
By the way, this line might be useless in the test helper... a plain string could be used rather than checking for an env variable...

Reply:
I'll do a minimal improvement.